website/public/locales/es/message.json

```diff
@@ -11,7 +13,7 @@
   "open_new_tab_action": "Abrir en una pestaña nueva",
   "parent": "Padre",
   "reactions": "Reacciones",
-  "recent_messages": "Mensajes recientes",
+  "recent_messages": "Mensajes recientes en {{language}}",
```

Comment:
Why adding the language to the recent messages translation?

Reply:
because thats how it should be:
https://github.com/LAION-AI/Open-Assistant/blob/71984331ffcca5f7386d212e2599ca1fce377c5f/website/public/locales/en/message.json#L16

Reply:
We have now filtering by language .. also the English string was change: `"recent_messages": "Recent Messages in {{language}}"`